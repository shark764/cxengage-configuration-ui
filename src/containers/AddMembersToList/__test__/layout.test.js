import React from 'react';
import { shallow } from 'enzyme';
import AddMemberToListLayout from '../layout';

describe('AddMemberToListLayout Layout', () => {
  it('thing', () => {});

  it('renders the AddMemberToListLayout', () => {
    const rendered = shallow(
      <AddMemberToListLayout
        onCancel={() => {}}
        userHasUpdatePermission={true}
      />
    );
    expect(rendered).toMatchSnapshot();
  });
});
