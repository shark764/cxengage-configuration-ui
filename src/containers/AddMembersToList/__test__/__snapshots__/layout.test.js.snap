// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMemberToListLayout Layout renders the AddMemberToListLayout 1`] = `
ShallowWrapper {
  "length": 1,
}
`;
